packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -71,16 +71,14 @@ export default abstract class BaseCall implements IWebRTCCall {
   public state: string = State[State.New];
 
   /**
-   * The `prevState` of the call.
-   * Usage with {@link State}:
+   * The previous state of the call.
+   * See {@link Call.state} for all possible values.
```

Comment:
Since we show docs for `Call`, which extends `BaseCall`, the link was not rendering properly